closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -1626,6 +1710,55 @@ JNIEXPORT jint JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_RSAEP
     return (jint)msg_len;
 }
 
+/* RSADP Cryptographic Primitive, RSA Private Key operation
+ * Returns -1 on error
+ * The param verify is -1 for 'no verify', otherwise it is size of m (with verify)
+ *
+ * Class:     jdk_crypto_jniprovider_NativeCrypto
+ * Method:    RSADP
+ * Signature: ([BI[BIJ)I
+ */
+
+typedef struct {
```

Comment:
Please give the `struct` a tag (`RSADP_BN`).

Reply:
Just wanted to make sure I understand what you meant by tag before resolving this; [this](https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/f50abc2/closed/src/java.base/share/native/libjncrypto/NativeCrypto.c#L1726) is what you meant right?